docker.Makefile

```diff
@@ -51,6 +52,10 @@ e2e-cross: create_bin
 	@$(call chmod,+x,bin/$(BIN_NAME)-e2e-darwin)
 	@$(call chmod,+x,bin/$(BIN_NAME)-e2e-windows.exe)
 
+dockerhub-publish:
+	docker build $(BUILD_ARGS) --target export-image -t $(DOCKERHUB_IMAGE_NAME):$(TAGNAME) .
+	docker push $(DOCKERHUB_IMAGE_NAME):$(TAGNAME)
```

Comment:
The `Jenkinsfile` gates on `TAGNAME` being set, but a manual call will not. 

I suggest adding as the first line of the recipe:
```
if [ -z "$(TAGNAME)" ] ; then echo "No TAGNAME specified"; exit 1; fi
```
which will abort if that happens

Reply:
yep